### PR TITLE
feat: expose paraswap v6 decoded trades

### PR DESCRIPTION
**Pull request by alexshchur**
### Description:

Expose ParaSwap v6 decoded trades

### CODE REVIEW

File: dbt_subprojects/dex/models/_projects/paraswap/arbitrum/paraswap_v6_arbitrum_trades_decoded.sql [hunk 1]
```diff
@@ -6,7 +6,11 @@
     file_format = 'delta',
     incremental_strategy = 'merge',
     incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.blockTime')],
-    unique_key = ['call_tx_hash', 'method', 'call_trace_address']
+    unique_key = ['call_tx_hash', 'method', 'call_trace_address'],    
+    post_hook='{{ expose_spells(\'["arbitrum"]\',
+                                "project",
+                                "paraswap_v6",
+                                \'["eptighte", "mwamedacen"]\') }}'
```
[the remark this answers is not in the document]
Comment (alexshchur) [reply dbt_subprojects/dex/models/_projects/paraswap/arbitrum/paraswap_v6_arbitrum_trades_decoded.sql]: hey @Hosuke ,
i've tried to apply this new verbose format but I'm getting [this compilation error](https://github.com/duneanalytics/spellbook/actions/runs/12428035060/job/34698903789?pr=7359) now
Screenshot 2024-12-20 at 10 30 10

I've checked configs and we're not using the deprecated field "tests" and use "data_tests" instead.
Do you know what might be the cause?

**Comment by 0xRobin** [comment]
please merge in `main` to resolve the compile issue

**Comment by alexshchur** [comment]
> please merge in `main` to resolve the compile issue

That worked, thank you.

Is all good to merge then?

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @alexshchur

**Pull request merged**